go/proxyd/config.go

```diff
@@ -72,6 +75,92 @@ type Config struct {
 	BackendGroups     BackendGroupsConfig `toml:"backend_groups"`
 	RPCMethodMappings map[string]string   `toml:"rpc_method_mappings"`
 	WSMethodWhitelist []string            `toml:"ws_method_whitelist"`
+	LogFormat         string              `toml:"log_format"`
+}
+
```

Comment:
I added some methods to the `Config` so that the code regarding building backends can be reused. Any strong preference on it being a method on the config vs a function that accepts the config?

Reply:
Function that accepts, the config, IMO. That way we can separate the business logic of building backends from the config itself.